src/web/modules/transactions/transactions.http.ts

```diff
@@ -3,11 +3,15 @@
 /* eslint-disable import/prefer-default-export */
 import { Request, Response, NextFunction } from 'express'
 
+import { Moment } from 'moment'
```

Comment:
It looks like this is both importing and requiring moment? 

Require is the only one used here and could be a const.

Reply:
Done